package.json

```diff
@@ -59,8 +53,35 @@
       "yarn format"
     ]
   },
-  "peerDependencies": {
-    "@nebula.js/stardust": ">=2.13.0"
+  "prettier": "@qlik/prettier-config",
+  "eslintConfig": {
+    "env": {
+      "browser": true
+    },
+    "extends": [
+      "@qlik/eslint-config",
+      "@qlik/eslint-config/jest"
+    ],
+    "rules": {
+      "arrow-parens": 0,
+      "comma-dangle": 0,
+      "implicit-arrow-linebreak": 0,
+      "max-len": 0,
+      "no-bitwise": 0,
+      "no-nested-ternary": 0,
+      "no-param-reassign": 0,
+      "no-plusplus": 0,
+      "no-restricted-globals": 0,
+      "no-unused-expressions": 0,
+      "object-curly-newline": 0,
+      "operator-linebreak": 0,
+      "import/no-extraneous-dependencies": [
+        2,
+        {
+          "devDependencies": true
+        }
+      ]
```

Comment:
A lot of rules here. We should check if we can remove some of these rules

Reply:
Agree. The value of a shared config kind of gets diminished if that many rules are overwritten